src/components/TeacherComponents/RegistrationAside/ResgistrationAside.tsx

```diff
@@ -0,0 +1,109 @@
+import { PropsWithChildren } from "react";
+import {
+  OakBox,
+  OakFlex,
+  OakHeading,
+  OakIcon,
+  OakLI,
+  OakSvg,
+  OakUL,
+} from "@oaknational/oak-components";
+
+import CMSImage from "@/components/SharedComponents/CMSImage";
+import { getIllustrationAsset } from "@/image-data";
+
+function ListItem({ children }: PropsWithChildren) {
+  return (
+    <OakLI
+      $display="flex"
+      $flexDirection="row"
+      $alignItems="center"
+      $gap="space-between-s"
+      $mb="space-between-xs"
+    >
+      <OakIcon iconName="tick" $height="all-spacing-7" $width="all-spacing-7" />
```

Comment:
I think a nicer option here, especially since this is rendered as a list, is for the alt text to be left empty `alt=""`  instead of `alt="tick"` since the icon is more of a decorative element and doesn't offer much value if a screen reader reads "tick" multiple times. If this is a component that cannot be easily adjusted just for this context, it's fine as it is.